Services/DataX.Flow/DataX.Flow.DeleteHelper/ConfigDeleter.cs

```diff
@@ -99,8 +99,8 @@ public async Task<ApiResult> DeleteFlow(JObject jObject)
             _inputEventhubConnectionStringRef = Helper.IsKeyVault(diag.EventhubConnectionString) ? diag.EventhubConnectionString : Helper.GenerateNewSecret(_keySecretList, _engineEnvironment.EngineFlowConfig.SparkKeyVaultName, ConfigName + "-input-eventhubconnectionstring", _engineEnvironment.EngineFlowConfig.SparkType, diag.EventhubConnectionString, false);
             diag.EventhubConnectionString = _inputEventhubConnectionStringRef;
 
-            if (diag.InputType == Constants.InputType_EventHub)

```

Comment:
Can this be unit tested?

Reply:
Added some unit tests for the helper functions in ConfigDeleter. For E2E, we should use scenario tester.